### DISCUSSION TITLE
How to disable the caption of the featureImage?

### DISCUSSION THREAD

**Discussion by konsumschaf** [chipzoller/hugo-clarity#149] [Q&A]
Hi all,

there was a change that added the Title as the alt text to the featureImage - that is very useful and seems to be seo relevant. But now the Title shows up under the first picture of an article as well - is there any way to disable this?

Thanks a lot

Thomas

**Comment by onweru** [comment]
Thanks, for reporting this @konsumschaf. Now fixed in #151
